.pre-commit-config.yaml

```
@@ -68,7 +68,7 @@ repos:
         args:
           # Indent by two spaces
           - -i
-          - '2'
+          - "2"
```

Unless necessary for the specific repository's configuration these files should generally only be changed in upstream skeleton repositories.

```suggestion
          - '2'
```

---

@mcdonnnj resolved with [25eefd9](https://github.com/cisagov/windows-server-packer/pull/22/commits/25eefd9d5266a223d81c3db85a307b46c8779641)